isstools/xasdata/export.py

```diff
@@ -0,0 +1,151 @@
+import uuid
+from databroker import Broker
+import time
+import numpy as np
+import os
+import os.path as op
+import jsonschema
+from event_model import DocumentNames, schemas
+
+
+from pathlib import Path
+
+# setup of objects to write to databroker
+
+
+# 1. Need handlers for external data
+# writing a file writer this is just an example
+# writing a file handler is here
+from databroker.assets.handlers import NpyHandler
+
+# and define the writer to write to file and return some descriptor
+
+# an example of writing to databroker 
+# db is the databroker object
+# db.mds is the metadatastore
+def store_results_databroker(md, data, db, filepath, root=None):
```

Comment:
This could be used as a prototype for `ingest` for iss processed files
We could create a repo called `suitcase-xafs`
(note this is different from `suitcase-pizzabox` which are raw pizza box files)